pyproject.toml

```
@@ -101,6 +101,7 @@ all_extras = [
   "scikit_posthocs>=0.6.5",
   "seaborn>=0.11",
   "seasonal",
+  'skforecast<=0.12.1,>=0.11; python_version < "3.12"',
```

This specification is not recommended. It's format in general should be greater than or equal to (inclusive) minimum (officially) supported stable (usually minor) version, and less than (exclusive) next upcoming stable version (again, usually minor).

Since we are adding it now, we should use latest version as minimum.

```suggestion
  'skforecast<0.13,>=0.12.1; python_version < "3.12"',
```

---

@fkiraly / @benHeid please share your comments here too.

---

Understood, I've made this change in my latest commit.